runtime/doc/pi_netrw.txt

```diff
@@ -440,11 +440,11 @@ settings are described below, in |netrw-browser-options|, and in
 
  *g:netrw_use_errorwindow* =2: messages from netrw will use a popup window
 			     Move the mouse and pause to remove the popup window.
-			     (default value if popup windows are availble)
+			     (default value if popup windows are available)
```

Comment:
We don't have 'em anyway, but these files are more-or-less taken verbatim, so :shrug:

Reply:
If we have docs for nonexistent features, we should also have nonexistent typos ;)